runelite-client/src/main/java/net/runelite/client/plugins/timetracking/clocks/ClockManager.java

```diff
@@ -57,7 +57,12 @@
 	private final List<Stopwatch> stopwatches = new ArrayList<>();
 
 	@Getter
-	private ClockTabPanel clockTabPanel = new ClockTabPanel(this);
+	private ClockTabPanel clockTabPanel;
+
+	ClockManager()
+	{
+		SwingUtilities.invokeLater(() -> clockTabPanel = new ClockTabPanel(this));
```

Comment:
This should be separate commit, and also explain why are you doing this

Reply:
The PR i linked had this change as it was causing a component creation off EDT issue for this file. It looks like this is no longer an issue though as I reverted this change and am not getting the error anymore. I don't understand why it's no longer an issue but I've reverted this change